src/utils/storage.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import LOCAL_STORE, { STARTED_KEY, LOCKED_KEY, CLOSED_KEY, IS_BACKUP_DONE_KEY, IS_HARDWARE_KEY } from '../storage';
+
+const storageUtils = {
+  /**
+   * Migrate the localStorage from a previous version to the current version.
```

Comment:
Was it a bug before? It's not in the acceptance criteria

Reply:
I tested the migration method, which migrates the access data to the current version, but during the QA the first screen to show when upgrading the version was the welcome screen since the `wallet:started` key was set to true but the new key was not.

So the issue was that we didn't even get to the locked screen to migrate the data.
Knowing this, I created this method to run during startup, a one time migration of keys which will take the user to the locked screen.